docs/podman-create.1.md

```diff
@@ -464,6 +477,13 @@ interactive shell. The default is false.
 Note: The **-t** option is incompatible with a redirection of the podman client
 standard input.
 
+**--uidmap**=map
+   UID map for the user namespace.  Using this flag will run the container with user namespace enabled, unless the --userns=host flag is turned on.
+
+   The following example maps uids 0-2000 in the container to the uids 30000-31999 on the host and gids 0-2000 in the container to the gids 30000-31999 on the host.
```

Comment:
Move example down to examples section of makefile

Reply:
same issue, the EXAMPLES section is not present

Reply:
Ok lets add one.

podman run definitely has one.